using System.Globalization;
using System.IO;
+using RL2.ModLoader;
using SceneManagement_RL;
using UnityEngine;

@@ -40,4 +_,5 @@

        Application.focusChanged += OnFocusChanged;
        ConfineMouseToGameWindow(!SaveManager.ConfigData.DisableCursorConfine);
+       ModLoader.LoadMods();
    }
